src/sparql/TripleBuilder.ts

```diff
@@ -101,6 +103,23 @@ export default class TripleBuilder {
 		return items;
 	}
 
+	public buildTripleForFilter( condition: Condition ): Triple {
```

Comment:
Looking at the triple, it doesn't look like it's a triple for Filter. It looks like a reference triple ("wasDerivedFrom"). What about naming it "buildReferenceTriple"?

Reply:
I just named it the first thing that came to my mind, yep `buildReferenceTriple` sounds better, will update.